### DISCUSSION TITLE
Anatomy of a Crypto Scam - Thomas Harris

### DISCUSSION THREAD

**Discussion by giscus** [maxpou/maxpou.fr#209] [General]
# Anatomy of a Crypto Scam - Thomas Harris

Thomas Roberts is a fraudulent YouTuber who uses fake crypto tutorials to spread malware. He artificially inflates his channel's popularity and promotes a malicious script that downloads harmful files and steals crypto. This scam, powered by AI-generated content, highlights the growing sophistication of online threats.

https://www.maxpou.fr/blog/crypto-scam-with-ai/

**Comment by Warciate** [comment]
hello thanks for this brilliant post, I fell in the trap what to do now ? is it that bad or ?

**Reply by maxpou** [reply]
Hi @Warciate, thanks for the kind words.

Yes, it's bad.
According to the "Hidden Script", here are a few things you can do:
* Scan your computer with an antivirus;
* Remove the registry entry that automatically starts `client32.exe`. You can do this by searching for "regedit" in the Start menu, navigating to `HKCU:\Software\Microsoft\Windows\CurrentVersion\Run`, and deleting the "NsSupport" value;
* Consider changing your passwords.

If you're not comfortable with 1&2, you should consider doing a hard reset + reinstall windows.

Good luck.

PS: btw, have you lost something?

**Reply by Warciate** [reply]
nono I mean its my school laptop so i dt have anything to protect on this computer except password like gmail etc but nothing with crypto or money

fuck this, that was sophisticated

**Comment by M4SQUER4ID** [comment]
Nice article, thanks (you've an "unintended" typo @Step 2 - should be unindented)

Ran this on a sandbox and it looks like cleanup just needs removal of the registry key and deletion of the %appdata%\Ns folder (provided you've not restarted or have one of the targeted wallets!)

No idea what the exe in the \Ns folder is up to ... yet!

**Reply by maxpou** [reply]
Thanks! I just corrected the typo :)

**Comment by kevin-bruton** [comment]
I guess I was stupid enough to run the script as well. As well as the registry entry that you have to remove, you have to stop the program that it executes. In the Task Manager it appear as "Ns Client Services" or something like that. Then you have to delete the whole program folder that it installs in AppData\Roaming\Ns.
I hadn't actually noticed anything strange. But apart from the folders the script looks, for, I've got no idea what that program was reporting or doing to my machine!

**Reply by kevin-bruton** [reply]
I suspect that the "Ns Client Services" that are installed is actually NSClient++ which is a monitoring agent. The worrying part is that once this is connected to, say a Nagios server, whoever has access to that server, as well as being able to do monitoring checks, they will also be able to execute scripts in the remote machine (the one running the client). So it is potentially a backdoor for the hacker to execute whatever they want and whenever they want on the remote machine!
